nilearn/datasets/atlas.py

```diff
@@ -59,12 +59,18 @@ def fetch_atlas_difumo(dimension=64, resolution_mm=2, data_dir=None, resume=True
 
     Returns
     -------
-    data : sklearn.datasets.base.Bunch
+    data : :func:`sklearn.utils.Bunch`
         Dictionary-like object, the interest attributes are :
 
-        - 'maps': str, 4D path to nifti file containing regions definition.
-        - 'labels': Numpy recarray containing the labels of the regions.
-        - 'description': str, general description of the dataset.
+        - 'maps': :obj:`str`, 4D path to nifti file containing regions
+          definition. The shape of the image is
+          ``(104, 123, 104, dimension)`` where ``dimension`` is the
+          requested dimension of the atlas.
+        - 'labels': :class:`numpy.recarray` containing the labels of
+          the regions. The length of the label array corresponds to the
+          number of dimensions requested. ``data.labels[i]`` is the label
+          corresponding to map ``i``.
```

Comment:
```suggestion
          corresponding to volume ``i`` in the 'maps' image.
```

Feel free to take or leave this suggestion. I just thought "map `i`" implied that it was a list of images instead of a 4D image.